doc/manual/source/installation.rst

```diff
@@ -0,0 +1,412 @@
+Installation
+============
+
+System Requirements
+-------------------
+
+When choosing a system for Cascade, Make sure you have [TODO] of available
+memory and [TODO] of disk space for the application. 
```

Comment:
Won't this depend heavily on the zones? Might be difficult to create a general guideline.